aten/src/ATen/function_wrapper.py

```diff
@@ -93,8 +93,16 @@ def TypedDict(name, attrs, total=True):  # type: ignore
 m.def("${unqual_schema_string}");
 """)
 
+# NOTE[UnboxedOnly] Many of our codegen templates currently exist twice, once
+# in an _UNBOXEDONLY_ variant and once without _UNBOXEDONLY_. This is because
+# ops that are `use_c10_dispatcher: full` need different c++ code than ops
+# that aren't `use_c10_dispatcher: full` yet. The _UNBOXEDONLY_ variants
+# are for ops that aren't `use_c10_dispatcher: full` yet and those code templates
+# can be deleted once all ops are `use_c10_dispatcher: full`.
```

Comment:
Tell people they need to update both if you modify one?

Reply:
I think it may also be profitable to comment all the sites where you're doing `if` conditionals on `use_c10_dispatcher`